docroot/modules/custom/mass_content/src/EventsRendererOrgPages.php

```diff
@@ -0,0 +1,127 @@
+<?php
```

Comment:
A suggestion: use strict types by default in all new classes?

https://architecture.lullabot.com/adr/20211201-use-strict-types-in-php-code/

Reply:
Great suggestion, I've read it before but I forget it. This should raise a warning on PHPCS.

Reply:
Filed https://massgov.atlassian.net/browse/DP-24449